update.sh

```diff
@@ -3,13 +3,15 @@ set -e
 
 declare -A gpgKeys
 gpgKeys=(
+	[7.0]='1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763'
```

Comment:
We already have to manually maintain one file or the other until PHP 7 is released, and since the `Dockerfile` already deviates from the existing `Dockerfile`s _and_ will still be updated every time there's a version bump, we'd prefer to do that maintenance in the `Dockerfile` itself (since we'll have to do a full re-sync with 5.6 at release time anyhow).

Since 7.0 isn't an official release and since these changes don't actually scrape "latest version" information, I'd much rather have the script warn and continue, even more especially because this change is temporary while we wait for a real official release.

It's also worth noting that the current "release" tarball is hosted in `~ab` (a private user directory), so if any other user happens to create future alpha, beta, or release candidate builds, they'd likely be in a new location and require a `Dockerfile` change anyhow.

I really wish upstreams would treat alpha, beta, and RC releases on par with stable releases and let us fetch them via the same methods, but that's not always possible.